scripts/logs/analyse_logs.py

```diff
@@ -0,0 +1,29 @@
+import argparse
+
+from common.formats import FormatFactory
+from logs_analysis import AnalysesFactory, AnalysisContext
+
+
+def get_args():
+    arg_parser = argparse.ArgumentParser(description="Simple ekf logs analysis")
+    arg_parser.add_argument("log_file", type=str, help="File with logs from ekf")
+
+    return arg_parser.parse_args()
+
+
+def main():
+    args = get_args()
+
+    file_handler = FormatFactory.from_path(args.log_file)
+    logs = file_handler.import_logs(args.log_file)
+
+    analyses = AnalysesFactory.get_analyses()
+
+    analyses_context = AnalysisContext(logs)
+
+    for analysis in analyses:
+        analysis.run(analyses_context)
```

Comment:
This is somewhat silly remark, but let's try to avoid multiple form of `analyses` in the code as they blend together and their abbreviation is... somewhat not usable in code :P

Consider the following ideas:
```suggestion
    handler = FormatFactory.from_path(args.log_file)
    logs = handler.import_logs(args.log_file)
    ctx = StudyContext(logs)

    for s in StudyFactory.get_studies():
        s.run(ctx)
```

or

```suggestion
    handler = FormatFactory.from_path(args.log_file)
    logs = handler.import_logs(args.log_file)
    ctx = DataContext(logs)

    for e in EvalFactory.get_studies():
        e.run(ctx)
```

also possible: jobs, tasks, scans etc.

Reply:
I have chosen `evaluations`